### PR TITLE
Adding Career SC role to project submissions view

### PR DESCRIPTION
**Pull request by laheara143**
## Describe context / purpose for this PR
Adding the Career SC role to have access to the project submissions view
## Issue link
https://cyberteamportal.atlassian.net/browse/D8-1834
## Link to MultiDev instance

http://md-1834-accessmatch.pantheonsite.io

## Checklist for PR author
- [ ] I have checked that the PR is ready to be merged
- [ ] I have reviewed the DIFF and checked that the changes are as expected
- [ ] I have assigned myself or someone else to review the PR

### CODE REVIEW

**Comment by laheara143** [comment]
Need to confirm if Career SC should have access to editing the project submissions. This would be changed on the webform config

File: web/sites/default/config/default/webform.webform.project.yml [hunk 1]
```diff
@@ -799,6 +799,7 @@ access:
       - sc
       - lt
       - ra
+      - careers_sc
```
Comment (laheara143) [inline web/sites/default/config/default/webform.webform.project.yml:802]: This does not seem to be the correct file so this change will be reverted

**Comment by laheara143** [comment]
Not sure where the correct location is to add the role but here is where I looked around /admin/structure/webform/manage/project/access

**Comment by a-pasquale** [comment]
That is the correct place. All set!

**Pull request merged**